[2025-08-18 18:37:14,207] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2025-08-17T00:00:00+00:00 [queued]>
[2025-08-18 18:37:14,213] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2025-08-17T00:00:00+00:00 [queued]>
[2025-08-18 18:37:14,213] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 18:37:14,213] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-18 18:37:14,213] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 18:37:14,226] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2025-08-17 00:00:00+00:00
[2025-08-18 18:37:14,229] {standard_task_runner.py:52} INFO - Started process 2188047 to run task
[2025-08-18 18:37:14,232] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'cria_pasta', 'scheduled__2025-08-17T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/meu_primeiro_dag.py', '--cfg-path', '/tmp/tmpvhjarms9', '--error-file', '/tmp/tmpflbw8nvs']
[2025-08-18 18:37:14,233] {standard_task_runner.py:80} INFO - Job 38: Subtask cria_pasta
[2025-08-18 18:37:14,275] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2025-08-17T00:00:00+00:00 [running]> on host itoneptl
[2025-08-18 18:37:14,324] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2025-08-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-17T00:00:00+00:00
[2025-08-18 18:37:14,325] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-18 18:37:14,326] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/root/Documents/airflow/pasta"']
[2025-08-18 18:37:14,331] {subprocess.py:85} INFO - Output:
[2025-08-18 18:37:14,334] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-18 18:37:14,353] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=meu_primeiro_dag, task_id=cria_pasta, execution_date=20250817T000000, start_date=20250818T213714, end_date=20250818T213714
[2025-08-18 18:37:14,403] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 18:37:14,429] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
